src/main/java/com/infoshareacademy/mistrzowieVaBank/controller/MainController.java

```diff
@@ -0,0 +1,296 @@
+package com.infoshareacademy.mistrzowieVaBank.controller;
+
+
+import com.infoshareacademy.mistrzowieVaBank.dao.OrderDao;
+import com.infoshareacademy.mistrzowieVaBank.dao.WineDao;
+import com.infoshareacademy.mistrzowieVaBank.dto.CartInfo;
+import com.infoshareacademy.mistrzowieVaBank.dto.CustomerInfo;
+import com.infoshareacademy.mistrzowieVaBank.dto.WineInfo;
+import com.infoshareacademy.mistrzowieVaBank.entity.Wine;
+import com.infoshareacademy.mistrzowieVaBank.form.CustomerForm;
+import com.infoshareacademy.mistrzowieVaBank.repository.OrderRepository;
+import com.infoshareacademy.mistrzowieVaBank.service.WineListService;
+import com.infoshareacademy.mistrzowieVaBank.utils.Utils;
+import com.infoshareacademy.mistrzowieVaBank.validator.CustomerFormValidator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@Controller
+@Transactional
+public class MainController {
+
+    @Autowired
+    private OrderDao orderDAO;
```

Comment:
```suggestion
    private final OrderDao orderDAO;
```
wstrzykiwane pola powinny byc oznaczone jako final